### ISSUE TITLE
make DefaultSqlOperations non-Postgres

### ISSUE THREAD

**Issue by jrhizor** [airbytehq/airbyte#3208]
Right now the insertion process is Postgres-specific in `DefaultSqlOperations`.

We could make `DefaultSqlOperations` an abstract class and have a separate `PostgresSqlOperations` class. This would also involve setting up the JDBC destination to use the new `PostgresSqlOperations` we would create as part of this task.

**Comment by cgardens** [comment]
Should we just rename DefaultSqlOperations to PostgresSqlOperations? Or is there more to it than that?

**Comment by jrhizor** [comment]
I don't think so since other classes extend this. I think we should make it clear which pieces are generally useful across jdbc destinations and what is required for a specific destination.
